### ISSUE TITLE
Pendo Guide on iOS Aligned Bottom has margins (React Native)

### ISSUE THREAD

**Issue by ErickMaeda** [pendo-io/pendo-mobile-sdk#63]
**Platform + Version**
iOS 16.4

**SDK Version**
`"rn-pendo-sdk": "^2.22.1"`

**Framework**
```json
"react-native": "^0.71.8"
"@react-navigation/bottom-tabs": "^6.5.7",
"@react-navigation/drawer": "^6.6.2",
"@react-navigation/native": "^6.1.6",
"@react-navigation/stack": "^6.3.16",
```

**Describe the bug**
The guide with modal aligned to the bottom is not really aligned to the bottom.
If you are using an iOS device that contains a Safe Area View, the modal is not aligned on the bottom, it has some margin.

I tried removing everything from my code, design, etc... Just left Pendo, and it behaves the same.

**To Reproduce**
Steps to reproduce the behaviour:

1. Create a Guide with the template `Small Pop Up Layout`
2. Test on iPhone >= 10 (Can be on simulator as well)
3. Location -> Position on Page -> Bottom Aligned

**Expected behavior**
The modal should have the same behaviour on all these devices. (iPhone 8, iPhone SE, iPhone SE 2nd, iPhone X, iPhone 14, ...)

**Sample Code**
- No code is needed, just to activate the modal.

**Additional context**
- Works on Android

<img width="551" alt="iPhone 8" src="https://github.com/pendo-io/pendo-mobile-sdk/assets/12499837/c6b6d5f5-d84f-40e6-8446-dfef7a0e1a80">
<img width="542" alt="iPhone 14" src="https://github.com/pendo-io/pendo-mobile-sdk/assets/12499837/faf5a648-8807-4f80-a9c1-1471a5d206fc">
<img alt="Pendo Guide" src="https://github.com/pendo-io/pendo-mobile-sdk/assets/12499837/7df83be0-794d-48d5-b95e-696ef573e7dc">

**Comment by MikePendo** [comment]
Hi @ErickMaeda 
Let us look in to that.( @naorlevi is it bug or by design)

**Comment by ErickMaeda** [comment]
Hello @MikePendo 

Do you have any update on it? Thank you!

**Comment by MikePendo** [comment]
@ErickMaeda 
Unfortunately no, its in our backlog and our PM need to priorotize the issue (as its not very crucial right now).
Its a little bit beyond my power to push it further 
@naorlevi

**Comment by simonpendo** [comment]
@ErickMaeda Can you please open a ticket to Pendo support ? they will help you or align with our PM

**Comment by MaximShnirman** [comment]
@ErickMaeda im currently closing the issue, in case it persists, please open a ticket at Pendo support

**Comment by 0xkuj** [comment]
was fixed in our next release 3.2, please wait for this to be released and upgrade
